repl/src/main/scala/ammonite/repl/interp/Interpreter.scala

```diff
@@ -97,18 +99,27 @@ class Interpreter(prompt0: Ref[String],
                 .combinePrints(${printSnippet.mkString(", ")})
         """,
         printer,
+        fileName,
         extraImports
       )
 
   }
 
-  def processModule(code: String) = processScript(
+  def processModule(code: String, fileName: String = "Main.scala") = processScript(
     skipSheBangLine(code),
-    (code, imports) => withContextClassloader(eval.processScriptBlock(code, imports, printer))
+    (code, imports) => withContextClassloader(eval.processScriptBlock(code,
+                                                                      imports,
+                                                                      printer,
+                                                                      fileName))
   )
 
+
   def processExec(code: String) =
-    processScript(skipSheBangLine(code), { (c, i) => evaluateLine(c, Nil, printer, i)})
+    processScript(skipSheBangLine(code), { (c, i) => evaluateLine(c,
```

Comment:
And let's format this as 

```scala
processScript(
  skipSheBangLine(code), 
  { (c, i) => evaluateLine(c, Nil, printer, "Main.scala", i) }
)
```

In general, try to avoid vertical alignment everywhere except class/function signatures. If method/function calls or assignments or other expressions are too long, 

- Kick them to the next line
- Indent the wrapped lines by 2
- Add the necessary closing paren/curly on a new line, not indented.

Basically try to format them about the same as you'd format the multi-line bodies of functions or classes: 2-space indent, one-thing-per-line inside, closing bracket on new line.